src/main/java/games/strategy/engine/lobby/server/db/UserController.java

```diff
@@ -64,16 +65,29 @@ public void updateUser(final DBUser user, final HashedPassword hashedPassword) {
 
     try (final Connection con = connectionSupplier.get();
         final PreparedStatement ps =
-            con.prepareStatement("update ta_users set password = ?,  email = ?, admin = ? where username = ?")) {
+            con.prepareStatement("update ta_users set password = ?,  email = ? where username = ?")) {
       ps.setString(1, hashedPassword.value);
       ps.setString(2, user.getEmail());
-      ps.setInt(3, user.isAdmin() ? 1 : 0);
-      ps.setString(4, user.getName());
+      ps.setString(3, user.getName());
       ps.execute();
       con.commit();
     } catch (final SQLException e) {
-      throw new IllegalStateException(
-          "Error updating name:" + user.getName() + " email: " + user.getEmail() + " pwd: " + hashedPassword.mask(), e);
+      throw new IllegalStateException(String.format("Error updating name: %s email: %s pwd: %s",
+          user.getName(), user.getEmail(), hashedPassword.mask()), e);
+    }
+  }
+
+  public void makeAdmin(final DBUser user, final boolean admin) {
```

Comment:
`DBUser` already carries a role indicating if the user is an admin or not.  Couldn't we just reuse that instead of adding the extra boolean?

Also, this public method is missing a Javadoc.